aten/src/ATen/preprocess_declarations.py

```diff
@@ -217,8 +217,20 @@ def signature(option, i=None, value=None):
                                                                        (raw_args - filtered_args)]
 
 
+def is_internal_method(option):
+    if 'method' in option['variants']:
+        return False
+    elif option.get('deprecated', False):
+        return False
+    elif not option['variants']:
+        return False
+    else:
+        return True
+
+
 def run(declarations):
     declarations = [d for d in declarations if not exclude(d)]
+    non_internal_method = set()
```

Comment:
nit: non_internal_methods

Reply:
also, can you add a comment explaining why you need this?

Reply:
final note (maybe you want to file an issue about this if you want to defer to a later patch): tensor is showing up in Type, but it should probably be TypeInternalInterface (we have at::empty for the equivalent).